src/Statistics.jl

```diff
@@ -1144,6 +1144,15 @@ end
 
 ##### Sampling #####
 
-function sample end
+"""
+    sample([rng], a)
+
+Uniformly select a single random element of `a`.
+
+Optionally specify a random number generator `rng` as the first argument
+(defaults to `Random.GLOBAL_RNG`).
+"""
+sample(rng::AbstractRNG, a::AbstractArray) = a[rand(rng, 1:length(a))]
```

Comment:
```suggestion
sample(rng::AbstractRNG, a::AbstractArray) = a[rand(rng, eachindex(a))]
```
Shouldn't this be `eachindex`?

Reply:
it should - exactly the same now occurred to me that we have a bug in StatsBase.jl.